src/js/game/LevelMVC/LevelView.js

```diff
@@ -1971,6 +1978,7 @@ module.exports = class LevelView {
         sprite = group.create(xOffset + 40 * x, yOffset + group.yOffset + 40 * y, atlas, frame);
         frameList = Phaser.Animation.generateFrameNames("Lava_", 0, 5, "", 0);
         sprite.animations.add("idle", frameList, 5, true);
+        this.psuedoRandomTint(group, sprite, x, y);
```

Comment:
why are these handled specially?

Reply:
Because this switch method has lots of duplicated code? 😄

It would be worth refactoring `createBlock` at some point to clean it up.

Reply:
Hah, fair. But still, why are these specifically handled specially, when it doesn't look like the old approach was treating them special?

Reply:
Ah, sorry, I see what you're asking!  The old approach indiscriminately place `Shade_Layer.png` over the entire ground group, regardless of block type.  To duplicate that effect we have to apply the tint to all ground block types.

Reply:
ahh, and these two are the ones that can end up as just regular ground blocks but which aren't caught by the `default`, gotcha.

In that case, we probably want to also do this for "watering"

Reply:
Followed up offline.  This actually calls back into the same function (`createBlock`) with "farmlandWet", which falls into the default case below.

https://github.com/code-dot-org/craft/pull/293/files#diff-894dfbadc994764b5b510d470f495021L1960